packages/workflow-core/src/lib/utils/context-transformers/qj-transformer.ts

```diff
@@ -1,19 +1,19 @@
-import { run } from 'node-jq';
-import { BaseContextTransformer, TTransformationLogic } from './types';
-import { TContext } from '../types';
-
-export class JQTransformer extends BaseContextTransformer {
-  name = 'jq-transformer';
-  mapping: TTransformationLogic;
-
-  constructor(mapping: TTransformationLogic) {
-    super();
-    this.mapping = mapping;
-  }
-
-  async transform(context: TContext, options: {}) {
-    const response = await run(this.mapping, context, options);
-
-    return response;
-  }
-}
+// import { run } from 'node-jq';
```

Comment:
Do we still need this file?

Reply:
not sure about it. might be injected later on. 
for now i see no harm of keeping it as a snippet